testdata/.tranor/envs.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
Is this testdata/.tranor what will be installed on the user's home by default when installing tranor? Shouldn't it be defined somewhere else with, perhaps, a link to it on testdata?